app/src/utils/claim-utils.ts

```diff
@@ -66,7 +66,8 @@ export const displayClaimValue = (claim: ClaimWithValue): string => {
   const formatters: { [key in ClaimType]: (value: any) => string } = {
     AdultCredential: (value: any) => value.over18,
     BirthCredential: (value: any) => value.dob,
-    NameCredential: (value: any) => value.firstName + " " + value.lastName,
+    NameCredential: (value: any) =>
+      value.firstName + " " + value.middleName + " " + value.lastName,
```

Comment:
Lets fix this tech debt.

```suggestion
    NameCredential: (value: any) =>
      `${value.firstName} ${value.middleName} ${value.lastName}`,
```

Reply:
All good, I was just trying to keep with what we had but I will adjust to this. :) thanks!